.github/workflows/test-action.yml

```
@@ -18,7 +19,7 @@ jobs:
       statuses: none
     name: Install chart-releaser and test presence in path
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
```

```suggestion
    - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```

Dependabot is going to open a PR to do this.

---

that is intentional :)